public/index.html

```diff
@@ -75,15 +95,23 @@ <h2 class="title">Configure</h2>
 
                     <label>
                         <input type="checkbox" class="nes-checkbox" />
-                        <span>Open the source code, and find the .env file</span>
+                        <span
+                            >Open the source code, and find the .env file</span
+                        >
                     </label>
                     <label>
                         <input type="checkbox" class="nes-checkbox" />
-                        <span>Paste your Discord, Uptime Robot, and Giphy keys in the appropiate place</span>
+                        <span
+                            >Paste your Discord, Uptime Robot, and Giphy keys in
+                            the appropiate place</span
+                        >
                     </label>
                     <label>
                         <input type="checkbox" class="nes-checkbox" />
-                        <span>Go to skills/hears.js and start adding your own functionality!</span>
+                        <span
+                            >Go to skills/hears.js and start adding your own
```

Comment:
this is really weird, the `>` should not be on a separate line!
I'll check the configuration

Reply:
Yeah, it's the `--brackets-same-line` option. It defaults to false, and apparently doesn't want to listen to config files or CLI options. I started using Prettier on another project, and ran into the same issue.

Reply:
because it is `jsxBracketSameLine` in config. I'll make it listen :grin: 
Glad to see you liked that enough to add it somewhere else :blush:

Reply:
OK I think it is safer to leave it that way. What do you think?

Reply:
Yeah, it looks weird, but I get it. Oddly enough, I actually intuited my way to almost figuring out that this option was the culprit. I just didn't realize that's what this option did until I re-read it just now. :)

Let's leave it in.